modules/environment/README.md

```diff
@@ -15,6 +15,7 @@ ZSH Options
 | AUTO_RESUME | Resume existing jobs before creating a new job |
 | LONG_LIST_JOBS | Use the verbose list format by default |
 | NOTIFY | Report job status immediately instead of waiting for new prompt |
+| INTERACTIVECOMMENTS | Recogniz `#` as comments
```

Comment:
I like the initiative of adding this.

I'd make this line as:

    | INTERACTIVE_COMMENTS | Recognize comments starting with `#` |

or "Allow" instead of "Recognize". I'd use the same "INTERACTIVE_COMMENTS" name, and the same comment in the `init.zsh` file.